src/ts/components/seekbar.ts

```diff
@@ -344,8 +350,21 @@ export class SeekBar extends Component<SeekBarConfig> {
           player.pause('ui-seek');
         }
       }
+    };
 
+    this.onSeek.subscribe((sender) => {
+      // track seeking status so we can catch events from seek preview seeks
+      this.isUserSeeking = true;
+      // Notify UI manager of started seek
+      uimanager.onSeek.dispatch(sender);
+
+      if (isGroupPlaybackAPIAvailable(player) && player.groupPlayback.hasJoined()) {
+        setTimeout(() => handleSeekEvent(), 100);
```

Comment:
After discussing the delay handling offline with @hawk23 we have the feeling that this is not really needed. We don't see any drawback if we always start the suspension no matter if seek happened by scrubbing or just by tapping.